Tutorials/13_Geometry-Optimization/opt_helper/physconst.py

```diff
@@ -0,0 +1,3 @@
+pi = 3.14159265358979323846264338327950288
+bohr2angstroms = 0.52917720859
```

Comment:
This file can safely be deprecated to avoid duplication, since each of these are available through either Psi4 or NumPy/Python `math`.  Pi is available (to the same `float64` precision as the above line) with both `np.pi` and `math.pi`, and the physical constants are available with `psi4.constants.bohr2angstroms` and `psi4.constants.hartree2aJ`.